acceptance/testdata/detector/container/workspace/mock_app/run

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
What is the purpose of this file?

Reply:
Forgot to delete the file. Will do it i as part of the next commit.